Sources/InfomaniakCore/Networking/ApiFetcher.swift

```diff
@@ -136,24 +140,51 @@ open class ApiFetcher {
             )
     }
 
+    @available(*, deprecated, message: "Use perform with ValidServerResponse instead")
```

Comment:
Discussed: Should we use the breaking change opportunity to remove this function a well? -> No